src/property.ts

```diff
@@ -921,8 +921,8 @@ function Property$ensureInited(property: Property, obj: Entity): void {
 	if (!obj.__fields__.hasOwnProperty(property.name)) {
 		Property$pendingInit(obj, property, true);
 
-		// Do not initialize calculated properties. Calculated properties should be initialized using a property get rule.
-		if (!property.isCalculated) {
+		// Do not initialize calculated properties. Calculated properties should be initialized using a property get rule. Lists must always be initialized however.
+		if (!property.isCalculated || property.isList) {
```

Comment:
Question: Would this cause a calculation or default to start raising a change event where it wouldn't have previously?

Reply:
Default calculations would not have `isCalculated = true`. For actually calculated lists, those were not used/working at all, so I don't think there is any room for regression there.

Reply:
Gotcha. Yeah, that's good that there won't be a regression. But, is the behavior consistent between list and non-list calculated properties? Maybe this would cause a default to recalculate when a calculated list value was established for the first time? Just trying to think if there are any potential gotchas.

Reply:
Yeah it would trigger a change event on the list the first time it calculates.

Reply:
Had to revert back to the original solution which initializes the list from the calc rule in order to fix the initial calculation behavior.